kDrive/UI/View/Files/FileDetail/ShareLink/ShareLinkTableViewCell.swift

```diff
@@ -115,12 +117,15 @@ class ShareLinkTableViewCell: InsetTableViewCell {
                 date
             )
             shareLinkStackView.isHidden = false
+            fileShareLinkSettingTitle.alpha = 1
```

Comment:
note: not super found of playing with the alpha, but I cannot use the `.hidden` property as it breaks the stack view.